models/onboardconfig.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import (
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+)
+
+type OnboardNamespace struct {
+	EnableMonitoring bool
+	Name             string
+	Quota            string
+}
+
+type OnboardingRequest struct {
+	Env                string `yaml:",omitempty"`
+	Namespaces         []OnboardNamespace
+	ProjectDescription string `yaml:"project_description"`
+	TargetCluster      string `yaml:"target_cluster"`
+	TeamName           string `yaml:"team_name"`
+	Template           string
```

Comment:
What does this mean? Is this being used anywhere, if so for what? (same goes for the config in testcase)